modules/reactor/test/tests/meshgenerators/polygon_concentric_circle_mesh_generator/pin_hex_reporting_id.i

```diff
@@ -0,0 +1,52 @@
+[Mesh]
+  [pin]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '6 6 6 6 6 6'
+    background_intervals = 2
+    polygon_size = 0.63
+    polygon_size_style ='apothem'
+    ring_radii = '0.2 0.4 0.5'
+    ring_intervals = '2 2 1'
+    use_sector_id=true
+    use_ring_id=true
```

Comment:
so looking at this input without context, I can read every parameter and understand it, except these two. 

That s not usually a requirement since there is supporting documentation for the code the user may look at, but it'd be nicer not to have to